CMakeLists.txt

```diff
@@ -130,6 +132,23 @@ if(WIN32)
   list(APPEND RESOURCES ${CMAKE_BINARY_DIR}/generated/version.rc)
 endif()
 
+add_library(
+    target_generated
+    OBJECT
+    src/Settings/SettingsInfo.hpp
+    ${CMAKE_BINARY_DIR}/generated/SettingsInfo.cpp
+)
```

Comment:
I guess this caused the CI to fail. Why separate them as a library?

Reply:
[check this](https://github.com/cpeditor/cpeditor/pull/858#issuecomment-862454311)
So I move the generated sources into another target to apply different clang-tidy setting.

Reply:
@neko-para is this resolved?